packages/components/src/styles/mixins/_button.scss

```diff
@@ -84,10 +84,6 @@ $hds-button-size-props: (
   border-color: var(--token-color-border-primary);
   box-shadow: none;
   cursor: not-allowed;
-
-  &::before {
-    border-color: transparent;
-  }
```

Comment:
After deletion of this style:
* `::file-selector-button::before` no longer exists in the compiled CSS output (packages/components/dist/styles/@hashicorp/design-system-components.css)
* There are no Percy diffs